src/pipelinedefinition.cpp

```diff
@@ -69,9 +69,19 @@ Status PipelineDefinition::validate(ModelManager& manager) {
         return validationResult;
     }
 
-    notifier.passed = true;
     SPDLOG_LOGGER_DEBUG(modelmanager_logger, "Finished validation of pipeline: {}", getName());
```

Comment:
Here you log "Finished validation..." which is not true. We include getInputsInfo and getOutputs info as part of validation process now. Move it after notifier.passed = true; line.